pkg/apis/build/v1beta1/build_types.go

```diff
@@ -70,6 +70,8 @@ const (
 	TriggerInvalidImage BuildReason = "TriggerInvalidImage"
 	// TriggerInvalidPipeline indicates the trigger type Pipeline is invalid
 	TriggerInvalidPipeline BuildReason = "TriggerInvalidPipeline"
+	// OutputTimestampValueNotSupportedForBuild indicates that an unsupported output timestamp setting was used
+	OutputTimestampValueNotSupportedForBuild BuildReason = "OutputTimestampValueNotSupportedForBuild"
```

Comment:
```suggestion
	OutputTimestampValueNotSupported BuildReason = "OutputTimestampValueNotSupported"
```

The ForBuild is not needed, its clear this comes from a Build.